src/server/svr_jobfunc.c

```diff
@@ -4728,12 +4727,28 @@ start_end_dur_wall(void *pobj, int objtype)
 			}
 			break;
 
+		case 10:    /* end, wall */
```

Comment:
@vstumpf we should do something similar when only end, duration are given with a reservation request. pbs_rsub's man page says that 2 of the following 3 options are mandatory -R, -E and -D. Considering this statement, -E and -D should produce same result as -E and -lwalltime. What do you think?

Reply:
Looking at the pbs_rsub code, -D is the same as -lwalltime.